### ISSUE TITLE
Ongoing Improvement: Improve Accuracy of Results / Reduce Confabulation Rate

### ISSUE THREAD

**Issue by rmusser01** [rmusser01/tldw_server#103]
Title.

See `Docs/Citations_and_Confabulations.md`

**Comment by rmusser01** [comment]
Seems like the best thing currently short of fine-tuning/pre-training is asking a model specifically trained to identify confabulations.

**Comment by rmusser01** [comment]
Finetuning: 
- https://eugeneyan.com/writing/finetuning/

Evals:
- https://github.com/yanhong-lbh/LLM-SelfReflection-Eval
- https://eugeneyan.com/writing/evals/

LLM As Judge:
- https://arxiv.org/pdf/2404.12272
- https://arize.com/blog/breaking-down-evalgen-who-validates-the-validators/

Detecting Hallucinations using Semantic Entropy:
- https://www.nature.com/articles/s41586-024-07421-0
- https://github.com/jlko/semantic_uncertainty
- https://github.com/jlko/long_hallucinations
- https://arxiv.org/abs/2406.15927

Lynx/patronus
- https://www.patronus.ai/blog/lynx-state-of-the-art-open-source-hallucination-detection-model
- https://huggingface.co/PatronusAI/Llama-3-Patronus-Lynx-8B-Instruct-Q4_K_M-GGUF
- https://github.com/NVIDIA/NeMo-Guardrails/blob/develop/docs/user_guides/community/patronus-lynx.md
- https://github.com/NVIDIA/NeMo-Guardrails/blob/develop/examples/configs/patronusai/prompts.yml
- https://arxiv.org/html/2407.08488v1
-

**Comment by rmusser01** [comment]
https://github.com/EdinburghNLP/awesome-hallucination-detection

**Comment by rmusser01** [comment]
https://arxiv.org/abs/2407.16557

**Comment by rmusser01** [comment]
Reasoning
	https://arxiv.org/abs/2407.19813
	https://arxiv.org/abs/2408.06195
	https://arxiv.org/abs/2407.13481
Output length impacts on reasoning
	https://arxiv.org/abs/2407.19825

**Comment by rmusser01** [comment]
https://arxiv.org/abs/2406.10279
https://huggingface.co/spaces/vectara/Hallucination-evaluation-leaderboard

**Comment by rmusser01** [comment]
https://arxiv.org/pdf/2409.18475
https://cleanlab.ai/blog/trustworthy-language-model/

**Comment by rmusser01** [comment]
https://arxiv.org/abs/2410.02707

**Comment by rmusser01** [comment]
https://www.lycee.ai/blog/rag-ragallucinations-and-how-to-fight-them

**Comment by rmusser01** [comment]
https://github.com/lechmazur/confabulations/

**Comment by rmusser01** [comment]
https://llm-editing.github.io/

**Comment by rmusser01** [comment]
https://arxiv.org/abs/2410.22071

**Comment by rmusser01** [comment]
https://github.com/aws-samples/llm-based-advanced-summarization/blob/main/detect_attribution.ipynb
https://huggingface.co/vectara/hallucination_evaluation_model

**Comment by rmusser01** [comment]
https://github.com/confident-ai/deepeval/tree/99aae8ebc09093b8691c7bd6791f6927385cafa8/deepeval/metrics/hallucination
